# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kde-extra-cmake-modules/hotfix-max-python.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- extra-cmake-modules-5.86.0/find-modules/FindPythonModuleGeneration.cmake.vanilla	2022-09-16 13:24:57.462000000 +0200
+++ extra-cmake-modules-5.86.0/find-modules/FindPythonModuleGeneration.cmake	2022-09-16 13:25:05.530000000 +0200
@@ -162,7 +162,7 @@
 
 if (NOT GPB_PYTHON3_LIBRARY)
   set(_PYTHON3_MIN_VERSION 4)
-  set(_PYTHON3_MAX_VERSION 10)
+  set(_PYTHON3_MAX_VERSION 11)
 
   _find_python(3 ${_PYTHON3_MAX_VERSION}) # Canary check
 
